### DISCUSSION TITLE
Missing library libtimpi_opt.so.6 after update

### DISCUSSION THREAD

**Discussion by Flolaffel** [idaholab/moose#20137] [Q&A Getting Started] [answered]
I updated moose yesterday and ran into following issue when I tried running my project:

`./intro-opt: error while loading shared libraries: libtimpi_opt.so.6: cannot open shared object file: No such file or directory`

Running `ldd myproject-opt` gives the result `libtimpi_opt.so.6 => not found`

Please help me fix this. Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

Did you update both MOOSE and libmesh? Like you followed the instructions here?
https://mooseframework.inl.gov/getting_started/installation/index.html#update

Guillaume

**Reply by Flolaffel** [reply]
Ah, just forgot `make clobberall`. Thank you.
